app/assets/javascripts/components/story/ExpandedStory/index.js

```diff
@@ -47,7 +50,19 @@ export const ExpandedStory = ({
           onEdit={(newAttributes) => editStory(story.id, newAttributes)}
         />
       </div>
+      <div>
+        <ExpandedStoryRequestedBy
+          story={story}
+          users={users}
+          onEdit={(newAttributes) => editStory(story.id, newAttributes)}
+        />
 
+        <ExpandedStoryOwnedBy
+          story={story}
+          users={users}
+          onEdit={(newAttributes) => editStory(story.id, newAttributes)}
```

Comment:
I think here you can change this to 

```suggestion
          onEdit={(value) => editStory(story.id, {ownedBy: value})}
```


so that way you can use onEdit, passing only the value, like this: `onEdit(user)`. And maybe... you can think a way to remove the duplication, because, `ExpandedStoryOwnedBy` and `ExpandedStoryRequestedBy` components both seem pretty similar.

What do you think about that?

Reply:
Nice tip, I removed the select on another component and use on both! :)